invenio_communities/assets/semantic-ui/js/invenio_communities/members/invitations/InvitationResultItem.js

```diff
@@ -80,13 +80,27 @@ export class InvitationResultItem extends Component {
             </Grid.Column>
           </Grid>
         </Table.Cell>
-        <Table.Cell>
+        <Table.Cell
+          className="mobile-table-cell"
+          data-label={i18next.t("Status")}
+        >
           <Overridable id={`RequestStatus.${request.status}`}>
-            <span>{request.status}</span>
+            <div>
```

Comment:
this should be a separate component `RequestStatus` with it's own Overridable id, not "hidden" in another component, like it is done in line 87.
I would suggest renaming existing `RequestStatusIcon` to `RequestStatus` and extending it to contain the status name as well, instead of adding 5 new overridden components.